README.md

```diff
@@ -18,6 +18,10 @@ Make sure that there is a search template at
 `kagiso_search/search_results.html` in your templates folder
 (see a sample below).
 
+Add the following required configuration settings:
```

Comment:
according to your code, its required, and doesn't have a default.

https://github.com/Kagiso-Future-Media/kagiso_search/blob/master/kagiso_search/views.py#L33